src/components/Faq/Faq.js

```diff
@@ -257,6 +257,26 @@ const Faq = ({ currentItemData, navData, faqHtml, crumbs, lang }) => {
               </div>
             );
           })}
+
+          {!!currentItemData.resources && (
+            <div className="ContentBlocks">
+              {currentItemData.resources.length > 0 && (
+                <ResourceList title="Nuorodos">
+                  {currentItemData.resources?.map((resource, j) => {
+                    return (
+                      <ResourceListItem
+                        key={j}
```

Comment:
> React recommends that you do not use indexes as keys, since it could impact performance negatively and could lead to some unstable component behaviour.

Can we use something else?

Reply:
Using keys as indexes impacts performance on places where you removing or adding items to array. If we're only listing some kind of data it's fine.